catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -1160,9 +1160,12 @@ private SortByType createSortBy(Query query) {
             sortBy = new SortByType();
             SortPropertyType sortProperty = new SortPropertyType();
             PropertyNameType propertyName = new PropertyNameType();
-            String propName = query.getSortBy()
-                    .getPropertyName()
-                    .getPropertyName();
+
+            String propName = null;
```

Comment:
@bdeining - if the sortBy property is missing, we shouldn't set the sortBy element in the request at all.  With this change, you could have a SortBy element with an empty property which doesn't make sense.